tika-parsers/src/test/java/org/apache/tika/parser/recognition/ObjectRecognitionParserTest.java

```diff
@@ -68,22 +78,49 @@ public void jpegTesorflowTest() throws IOException, TikaException, SAXException
 
     @Ignore("Configure Rest API service")
     @Test
-    public void testREST() throws Exception {
-        try (InputStream stream = loader.getResourceAsStream(CONFIG_REST_FILE)){
+    public void jpegRESTObjRecTest() throws Exception {
+        String[] expectedObjects = {"Egyptian cat", "tabby, tabby cat"};
+        doRecognize(CONFIG_REST_FILE_OBJ_REC, CAT_IMAGE_JPEG,
+                ObjectRecognitionParser.MD_KEY_OBJ_REC, expectedObjects);
+
+
+    }
+
+    @Ignore("Configure Rest API service")
+    @Test
+    public void jpegRESTim2txtTest() throws Exception {
+        String[] expectedObjects = {"baseball", "bat", "field"};
```

Comment:
This is a week assertion. 
Since this is a caption, considering adding phrases such as `baseball player holding bat` instead of  just the object names